### DISCUSSION TITLE
Discussion of census disparity

### DISCUSSION THREAD

**Discussion by ReidWeb** [XIVStats/XIVStats#59] [General]
Starting a thread if people wish to discuss the issue detailed in #57

**Comment by konomikitten** [comment]
The Oceania servers are really new and I think the bar of:

> Any reference to "Active" characters, refers to characters that have claimed the following item: The minion & mount for completing the Endwalker Main Scenario Quest from the Patch 6.0 story

Really does not represent the actual amount of active characters on that data center. Is there any chance an exception can be made and Oceania's active requirement be dropped too completing ARR for a timeframe, for example 6 months to a 1 year?

**Reply by ReidWeb** [reply]
Fair feedback! Including Materia in this round was something we perhaps shouldn't have done. 

There's a suggestion on Reddit to rename "active" to "end game" which we'll probably follow through on

**Reply by konomikitten** [reply]
> There's a suggestion on Reddit to rename "active" to "end game" which we'll probably follow through on

That sounds great to me. Also I had another thought, it would be neat if we could see stats on which expansions have been completed as well.

**Reply by ReidWeb** [reply]
Yep we can for sure look at that!
